client/verta/verta/_cli/registry/list.py

```diff
@@ -25,11 +30,28 @@ def lst():
 def lst_model(filter, output, workspace):
     """List all models available.
     """
-    click.echo("Listing all models")
+    client = Client()
+
+    models = client.registered_models.with_workspace(workspace).find(filter)
 
     click.echo("filter: {}".format(filter))
     click.echo("output: {}".format(output))
 
+    click.echo("result count: {}".format(len(models)))
+    click.echo("Listing all models")
```

Comment:
I realized that these echos are actually unnecessary, and not good as CLI output.
```suggestion
    click.echo()
```